test/extension-tests/successful-build/test/preferred-generators.test.ts

```diff
@@ -0,0 +1,211 @@
+import * as chai from 'chai';
```

Comment:
With the tests moved here, what of the `vs-preferred-gen` test directory?

Reply:
I would remove it but not in this review. The vs-preferred-gen test the configuration too.